packages/developer-portal/src/core/connect-session.ts

```diff
@@ -0,0 +1,12 @@
+import { ReapitConnectBrowserSession } from '@reapit/connect-session'
+// Referencing config direct rather than via window because window.reapit.config not available at compile time
+// and throws a runtime error
+import config from '../../config.json'
+
+// Needs to be a singleton as the class is stateful
+export const reapitConnectBrowserSession = new ReapitConnectBrowserSession({
+  connectClientId: config.cognitoClientId,
+  connectOAuthUrl: config.cognitoOAuthUrl,
+  connectLoginRedirectPath: '/welcome',
```

Comment:
Default route is `/apps` - would prefer if we use this and only redirect to welcome if the cookie is not set